### ISSUE TITLE
aws ec2 describe-reserved-instances / getting region

### ISSUE THREAD

**Issue by eugeneiohc** [aws/aws-cli#5347]
Confirm by changing [ ] to [x] below:
- [x] I've gone though the [User Guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) and the [API reference](https://docs.aws.amazon.com/cli/latest/reference/)
- [x] I've searched for [previous similar issues](https://github.com/aws/aws-cli/issues) and didn't find any solution

Issue is about usage on:
- [x] Service API : I want to do X using Y service, what should I do?
- [ ] CLI : passing arguments or cli configurations.
- [ ] Other/Not sure.

**Platform/OS/Hardware/Device**
Latest version of aws-cli

**Describe the question**
I am trying to get information about reserved instances and the current account that I'm working with has RI's that work within an entire region meaning the "Availability Zone" attribute is empty and the "Scope" is simply region. This is fine; however, I'm not sure how to get the region that these reserved instances are in without manually doing so. 

tl;dr I want to automatically find the region of a reserved instance that is not zone-specific via describe-reserved-instances or some other command.

**Comment by KaibaLopez** [comment]
Hi @eugeneiohc ,
Hmmm, if describe-reserved-instances "availability zone" returns empty... I'm not sure there is another way... could you try ```aws ec2 describe-regions --no-all-regions```
And let me know if that works for you?

**Comment by eugeneiohc** [comment]
> Hi @eugeneiohc ,
> Hmmm, if describe-reserved-instances "availability zone" returns empty... I'm not sure there is another way... could you try `aws ec2 describe-regions --no-all-regions`
> And let me know if that works for you?

Hi @KaibaLopez, 

I ended up using the --region tag to specify the region that I wanted. I figured that I could automate that script and then get the region that way. 

Thank you for responding regardless.
